sample/android/src/main/res/layout/item_picker.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto">
+
+    <data>
+        <variable
+            name="viewModel"
+            type="com.trikot.viewmodels.sample.viewmodels.PickerListItemViewModel" />
+
+        <variable
+            name="lifecycleOwnerWrapper"
+            type="com.mirego.trikot.viewmodels.LifecycleOwnerWrapper" />
+    </data>
+
+    <androidx.constraintlayout.widget.ConstraintLayout
```

Comment:
Is the contraint layout required here? We could reduce the overhead by exposing the <Spinner> directly has the root.